models/models.go

```diff
@@ -19,13 +21,23 @@ type Icon struct {
 // Icons is an array of icons
 type Icons []Icon
 
+// ErrorWithRecommendations ...
+type ErrorWithRecommendations struct {
+	Error           string
+	Recommendations step.Recommendation
+}
+
+// ErrorsWithRecommendations is an array with an Error and its Recommendations
+type ErrorsWithRecommendations []ErrorWithRecommendations
+
 // ScanResultModel ...
 type ScanResultModel struct {
-	ScannerToOptionRoot       map[string]OptionNode       `json:"options,omitempty" yaml:"options,omitempty"`
-	ScannerToBitriseConfigMap map[string]BitriseConfigMap `json:"configs,omitempty" yaml:"configs,omitempty"`
-	ScannerToWarnings         map[string]Warnings         `json:"warnings,omitempty" yaml:"warnings,omitempty"`
-	ScannerToErrors           map[string]Errors           `json:"errors,omitempty" yaml:"errors,omitempty"`
-	Icons                     []Icon                      `json:"-" yaml:"-"`
+	ScannerToOptionRoot               map[string]OptionNode                `json:"options,omitempty" yaml:"options,omitempty"`
+	ScannerToBitriseConfigMap         map[string]BitriseConfigMap          `json:"configs,omitempty" yaml:"configs,omitempty"`
+	ScannerToWarnings                 map[string]Warnings                  `json:"warnings,omitempty" yaml:"warnings,omitempty"`
+	ScannerToErrors                   map[string]Errors                    `json:"errors,omitempty" yaml:"errors,omitempty"`
+	ScannerToErrorsWithRecomendations map[string]ErrorsWithRecommendations `json:"errors_with_recommendations,omitempty" yaml:"errors_with_recommendations,omitempty"`
```

Comment:
Please add   yaml:"-"`   attribute to ignore this new field when running bitrise-init as a standalone binary (only used for debugging purpuses)

Reply:
This is not a big issue after all.

Reply:
Done ;)